README.md

```diff
@@ -76,6 +76,10 @@ Thanks for your interest in contributing!
 
 Review the [code of conduct](./CODE_OF_CONDUCT.md) and submit a pull request!
 
+# Community
+
+Unofficial Workspacer Matrix: https://matrix.to/#/#workspacer-community:matrix.org
```

Comment:
I think this can be refined slightly, but apart from that, I'm adding this.

`````md
You may join in our unofficial community chat hosted on the [matrix-platform](https://matrix.org/).

Our community can be found on [#workspacer-community:matrix.org](https://matrix.to/#/#workspacer-community:matrix.org).
`````